R/bayesianmetaanalysiscommon.R

```diff
@@ -1528,10 +1527,15 @@
       ggplot2::scale_color_manual("", values = c("slategrey", "black"), labels = c(gettext("Estimated"), gettext("Observed"))) +
       ggplot2::scale_shape_manual("", values = c(16, 15)) +
       ggplot2::guides(shape = ggplot2::guide_legend(reverse=TRUE, override.aes = list(size=3)), colour = ggplot2::guide_legend(reverse=TRUE)) +
-      ggplot2::theme(axis.text.y.right = ggplot2::element_text(colour = c(rep(c("black", "slategrey"), each = nrow(df)), rep("black", 3)))) +
-      ggplot2::xlab(bquote(paste(.(gettext("Effect size")), ~mu)))
+      ggplot2::theme(axis.text.y.right = ggplot2::element_text(colour = c(rep(c("black", "slategrey"), each = nrow(df)), rep("black", 3))))
   }
-  
+
+  plot <- plot +
+    ggplot2::scale_x_continuous(
+      name = bquote(paste(.(gettext("Effect size")), ~mu)),
+      breaks = jaspGraphs::getPrettyAxisBreaks(range(c(df$lower, df$upper))),
+      limits = range(jaspGraphs::getPrettyAxisBreaks(range(c(df$lower, df$upper)))))
```

Comment:
perhaps store `jaspGraphs::getPrettyAxisBreaks(range(df$lower, df$upper))` in a variable, rather than computing it twice? 
```suggestion
  xBreaks <- jaspGraphs::getPrettyAxisBreaks(range(df$lower, df$upper))
  plot <- plot +
    ggplot2::scale_x_continuous(
      name = bquote(paste(.(gettext("Effect size")), ~mu)),
      breaks = xBreaks,
      limits = range(xBreaks))
```
also, `range(a, b)` is identical to `range(c(a, b))`.